src/xenomods/modules/CameraTools.cpp

```diff
@@ -189,12 +189,15 @@ namespace xenomods {
 			fc->fov = std::clamp(fc->fov + -lStick.y * fovMult, -179.f, 179.f);
 			if (fc->fov == 0)
 				fc->fov = 0.001f;
+
+			move = { 0, rStick.y, 0 };
```

Comment:
Let the camera still move around with the left stick while this is going on, I feel like it'd be good for sweeping shots. Be sure to add a comment explaining what behavior this has, too. (Something along the lines of `// rise on Y-axis with right stick` but with a pinch more thought to it :P)